lh-dashboard/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
we are using pre-commit and not husky, should we keep husky? guess we do not 🤔

Reply:
let's leave it there

Reply:
tried with pre-commit before husky, however it has more complexity, even thought at some point we can double check this :) so we can have only 1 pre-commit tool